packages/react-components/src/components/RichTextEditor/RichTextSendBox.test.tsx

```diff
@@ -117,26 +117,3 @@ describe('RichTextSendBox should return text correctly', () => {
     expect(changedValue).toEqual(result);
   });
 });
-
-/* @conditional-compile-remove(rich-text-editor) */
```

Comment:
Rooster uses ::before pseudo element for placeholder text and it's not possible to access it from test here. Remove this test as we still check the placeholder in ui tests